flocker/acceptance/testtools.py

```diff
@@ -897,11 +901,39 @@ def release_all(leases):
                     Exception("Timed out cleaning up leases"),
                 )
 
+        def cleanup_volumes():
```

Comment:
:question: is this likely to increase test runtime as the setup is doing more work? If so be
on the lookout for increased test timeouts and increase the overall test runtime.